activitysim/examples/prototype_sandag_xborder/configs/tour_od_choice.yaml

```diff
@@ -55,3 +55,7 @@ SEGMENTS:
 
 LOGSUM_SETTINGS: tour_mode_choice
 LOGSUM_PREPROCESSOR: preprocessor
+
+compute_settings:
+  protect_columns:
+  - origin_destination
```

Comment:
This PR will break the sandag model that's not in the test system... are we keeping track of these breaking changes anywhere?  I think there have been a few of these since we started the Phase 8 work and I don't want them to get lost...